assets/js/optimizely.js

```diff
@@ -0,0 +1,98 @@
+(function($) {
+  // Instantiate an Optimizely client
+  const options = {
+    sdkKey: '8E1RuW2VLAS2csau84LZRq',
+    datafileOptions: {
+      autoUpdate: true,
+      updateInterval: 6000, // 1 minute in milliseconds
+    }
+  };
+  const optimizelyClientInstance = window.optimizelySdk.createInstance(options);
+
+  optimizelyClientInstance.onReady().then(() => {
+    // optimizelyClientInstance is ready to use, with datafile downloaded from the Optimizely CDN
+
+    const optimizelyEndUserIdCookieKey = 'optimizelyEndUserId';
+    if (!getCookiebyName(optimizelyEndUserIdCookieKey)) {
+      // Create optimizely UserID
+      setCookie(optimizelyEndUserIdCookieKey, generateUUID(), 365);
+    }
+    const userId = getCookiebyName(optimizelyEndUserIdCookieKey);
+
+    /**
+     * Experiment: Navbar vs Checkout Flow
+     * Description: This experiment seeks to test out the variations of navbar and checkout flows
+     * to track the following in each variation:
+     * How many users;
+     *
+     * - End up on the checkout page
+     * - Click the checkout button
+     * - End up on the contact page
+     * - Contact us after reaching the contact page
+     * - Click on the other buttons in the Nav
+     * - Pick the add ons per cloud provider
+     * - Click the learn more vs Get Demo
+     * - Click ask a grunt vs contact sales
+     */
+    (function () {
+      var variation = optimizelyClientInstance.activate('navbar_and_checkout_flow', userId);
+      if (variation === 'original_nav_plus_original_checkout_flow') {
+        // execute code for original_nav_plus_original_checkout_flow
+        $('#navbar-original').show();
+        $('#learn-more').show();
+      } else if (variation === 'original_nav_plus_beta_checkout_flow') {
+        // execute code for original_nav_plus_beta_checkout_flow
```

Comment:
Its because this variation really shouldn't work because of the issues i described in https://github.com/gruntwork-io/gruntwork-io.github.io/pull/129#issuecomment-516500571
I will remove it entirely.